docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -409,6 +409,20 @@ def my_asset(context):
     return context.resources.api.fetch_table("my_asset")
 ```
 
+Like ops, configuration is also supported for assets. Configuration is accesible through the asset context at runtime and can be used to specify behavior. Note that asset configuration behaves the same as configuration for ops.
```

Comment:
nitpick: accessible spelling

Reply:
Not a nitpick at all, thanks for catching this!